src/SFA.DAS.AANHub.Application/SFA.DAS.AANHub.Application.csproj

```diff
@@ -6,6 +6,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Profiles\Queries\GetProfiles\GetProfilesQueryResult.cs" />
```

Comment:
Delete the result file, you have excluded it, hence it is in the csproj and also the file is in the PR